app/src/main/java/org/hyperledger/besu/controller/QbftBesuControllerBuilder.java

```diff
@@ -234,7 +234,8 @@ protected MiningCoordinator createMiningCoordinator(
 
     // NOTE: peers should not be used for accessing the network as it does not enforce the
     // "only send once" filter applied by the UniqueMessageMulticaster.
-    peers = new ValidatorPeers(validatorProvider, Istanbul100SubProtocol.NAME);
+    // Use Peers instead of ValidatorPeers to allow message relay by non-validators
+    peers = new Peers(Istanbul100SubProtocol.NAME);
```

Comment:
My main concern is there will be an increase in duplicate messages and this could fill the event queue. I think might need to do de-duplication earlier before the message is added to the event queue.